script/Ciel/Day 14/14_01B_03_CIEL14B_4_1_2.txt

```diff
@@ -512,17 +568,18 @@ Now that she's seen by him this much, even he would be disgusted with her.
 [sha:dd38d875f515ea2a25b35e615d6e92810ad5f496]{
 -- Page 167, Offset 40741.
 -- ―――何もかも、手遅れなんだ”』
-―――It was too late for anything."
+It's too late for anything, now."
 }
 [sha:6ed87fe6ece1aa4dbaf198d3a053eab284142659]{
 -- Page 168, Offset 40742.
 -- 消える。消える。
-She will vanish.
+She'll vanish.
+ She's going to vanish.
 }
 [sha:03c6c6646c006d61d903836d2fb0258c373ecb81]{
 -- Page 168, Offset 40743.
 -- アルクェイド・ブリュンスタッドという意識が消える。
-The consciousness of Arcueid Brunesteid will vanish.
+The consciousness known as Arcueid Brunestud will vanish.
```

Comment:
/Essentially, she'll be replaced by "something" (Crimson Moon)
She won't actually disappear as in like, explode. But the Arcueid persona will disappear. It's part of mats
>Arcueid Brunestud's individuality will disappear.